modules/programs/fzf.nix

```
@@ -88,6 +88,23 @@ in {
       '';
     };
 
+    tmux = {
+      enableShellIntegration = mkEnableOption ''
+        setting <literal>FZF_TMUX=1</literal> which causes shell integration to use fzf-tmux
+      '';
+
+      shellIntegrationOptions = mkOption {
+        type = types.listOf types.str;
+        default = [ ];
+        example = [ "-d 40%" ];
```

```suggestion
        example = literalExample ''[ "-d 40%" ]'';
```
One last thing to fix.